zanata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<config xmlns="http://zanata.org/namespace/config/">
+  <url>https://translate.engineering.redhat.com/</url>
+  <project>CloudForms_UI</project>
+  <project-version>4.0</project-version>
```

Comment:
And the version should correlate to the upstream version (Capablanca).  /cc @matthewd @Fryguy